xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -1006,6 +1006,29 @@ rebuild_master:
             </para>
           </note>
         </sect4>
+
+        <sect4 xml:id="sec-obs-obs-scm-ci-workflow-integration-obs-workflows-filters-labels-filter">
+          <title>Labels Filter</title>
+
+          <para>The labels filter attempts to match labels to the pull request label. The workflow will
+          run only if the pull request label matches one of the labels in the filter.</para>
+
+          <para>Example of running a workflow for a <emphasis>duplicate</emphasis> label:</para>
+
+          <screen>workflow:
+  steps:
+    - branch_package:
+        source_project: home:jane_doe
+        source_package: ctris
+        target_project: games
+  filters:
+    labels:
+      only:
+        - duplicate </screen>
+
```

Comment:
```suggestion
<para>Removing the label matching the one's defined in the filter section from the pull request will do the following: </para> 
```

Reply:
Thanks. I've added the suggested changes